labrad/backend.py

```diff
@@ -45,6 +47,13 @@ def disconnect(self):
             self._disconnect()
             self.connected = False
 
+    def spawn(self):
+        """Start a new independent backend connection to the same manager."""
+        inst = self.__class__(name=self.name)
```

Comment:
Is the copying the name attribute important?  It might be clearer to make this a `@classmethod` and take the new connection name as a parameter (with the default value None which lets `__init__` choose).